src/topology/Top/stalks.lean

```diff
@@ -70,7 +70,13 @@ local attribute [tidy] tactic.op_induction'
   ℱ.stalk_pushforward C (𝟙 X) x = (stalk_functor C x).map ((pushforward.id ℱ).hom) :=
 begin
   dsimp [stalk_pushforward, stalk_functor],
-  tidy,
+  ext1,
```

Comment:
What is the reason for expanding `tidy` here? Speed?

Reply:
It is faster, but also this is a very fragile proof, relying on a lot of definitional equality under the hood. The proof broke quite mysteriously after making the other changes, and so I decided to make it much more explicit, and in particular use explicit rewrites instead of `simp`, for safety.

Reply:
Ok, understood. Let's hope it doesn't break again (-;
(Is the first `dsimp` still necessary? Would it make sense to make things a little bit more cleaner/idiomatic? E.g., by passing a name to `ext` and by using `rcases` on `j`.)